crates/bevy_text/src/error.rs

```diff
@@ -7,4 +7,6 @@ pub enum TextError {
     NoSuchFont,
     #[error("failed to add glyph to newly-created atlas {0:?}")]
     FailedToAddGlyph(GlyphId),
+    #[error("exceeded {0:?} availble TextAltases for font. This can be caused by using an excessive number of font sizes. Consider using Transform::scale to modify font size dynamically. If you need more font sizes modify TextSettings.max_font_atlases." )]
```

Comment:
Typo
```suggestion
    #[error("exceeded {0:?} available TextAltases for font. This can be caused by using an excessive number of font sizes. Consider using Transform::scale to modify font size dynamically. If you need more font sizes modify TextSettings.max_font_atlases." )]
```